readme.txt

```diff
@@ -2,6 +2,9 @@
 
 == Changelog ==
 
+= [4.11.6] TBD =
+
```

Comment:
We'll need to set something here if we're merging to master.

Reply:
Still TBD on what will actually be the Changelog. Will update once we determine  that, we will run with an In Branch QA